VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsBaseFragment.java

```diff
@@ -51,6 +49,7 @@ public abstract void onItemClick(DiscussionThread thread, AdapterView<?> parent,
     @Override
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
+        courseData = (EnrolledCoursesResponse) getArguments().getSerializable(Router.EXTRA_ENROLLMENT_COURSE_DATA);
```

Comment:
This should be initialized in `onCreate()`, not `onCreateView()`, since this is seed data and not bound to the view. It's unfortunate that there doesn't seem to be a RoboGuice annotation for injecting Fragment arguments though :disappointed: